Gems/ROS2/Code/Source/Lidar/ROS2LidarSensorComponent.cpp

```diff
@@ -17,19 +17,19 @@
 
 namespace ROS2
 {
-    namespace Internal
+    namespace
     {
         const char* kPointCloudType = "sensor_msgs::msg::PointCloud2";
     }
 
     void ROS2LidarSensorComponent::Reflect(AZ::ReflectContext* context)
     {
-        LidarSensorConfiguration::Reflect(context);
+        LidarBase::Reflect(context);
```

Comment:
This is not a good place to call it since ROS2Lidar2DSensorComponent also serializes this, possibly leaving it without reflected member (serialization error).

Move reflection to the system component where there is no single owner.